src/app/atoms/FormSubmitButton.tsx

```diff
@@ -39,10 +40,10 @@ export const FormSubmitButton: FC<FormSubmitButtonProps> = ({
   const otherProps = useMemo(() => (loading ? setAnotherSelector('loading', '') : null), [loading]);
 
   return (
-    <Button className={classNameMemo} disabled={disabled} {...rest} {...otherProps}>
+    <ButtonRounded className={classNameMemo} disabled={disabled} {...rest} {...otherProps}>
```

Comment:
You can pass isLoading to the RoundedButton, so it will show the loading spinner according to the new design